src/test/resources/sql/addressInsert.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO country (iso3166, name) VALUES ('036', 'Australia');
+INSERT INTO country (iso3166, name) VALUES ('040', 'Austria');
+INSERT INTO country (iso3166, name) VALUES ('031', 'Azerbaijan');
+INSERT INTO country (iso3166, name) VALUES ('060', 'Bermuda');
+INSERT INTO country (iso3166, name) VALUES ('804', 'Ukraine');
+
+INSERT INTO address (id, apartment_number, city, district, house_number, region, street, zipcode, description, country_iso3166)
```

Comment:
Do we have Id autogenerated? Can we omit to specify it and it will be generated automatically?

Reply:
I tried without id field, but it worked only when launched one method  

didn`t work without id when launch all tests
didn`t pass tests in class AddressControllerTest 
@Test
public void findByIdTest() {

java.lang.IllegalStateException: Expected response body to be verified as JSON, HTML or XML but no content-type was defined in the response.
Try registering a default parser using:
   RestAssured.defaultParser(<parser type>);
Content was:

and

@Test
public void createTest() {

java.lang.AssertionError: 1 expectation failed.
JSON path id doesn't match.
Expected: <5>
  Actual: 13


Could you give me some advice? if you know how to do it without id field

Reply:
Can you check if the database is populated with the ids if you remove them from the sql files?

Reply:
I tried without id field, but it worked only when launched one test method not for all.
I described it above.

example: class AddressControllerIT for all methods how it work in my opinion
First:
When started tests hibernate dropped and created tables

Hibernate: drop table if exists address CASCADE 
Hibernate: drop table if exists country CASCADE 
Hibernate: create table address (id bigint generated by default as identity, apartment_number VARCHAR(10), city VARCHAR(50), description TEXT, district VARCHAR(50), house_number VARCHAR(10), region VARCHAR(50), street VARCHAR(50), zipcode VARCHAR(15), country_iso3166 VARCHAR(3), primary key (id))
Hibernate: create table country (iso3166 VARCHAR(3) not null, name VARCHAR(255), primary key (iso3166))
Hibernate: alter table address add constraint FKm74r791055ljr738xrk15bwr foreign key (country_iso3166) references country

Second:
started method findAllTest() 
this script -> @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/addressInsert.sql") inserted data into tables country and address
table address will have next data:

id=1; apartment_number=1; city=Kiev e.t.c. 
id=2; apartment_number=4; city=Kiev e.t.c.
id=3; apartment_number=3; city=Bermuda e.t.c.
id=4; apartment_number=5; city=Azerbaijan e.t.c.

test for method findAllTest() = OK passed, will green
after that this script -> @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDrop.sql") removed data from tables country and address

Third: 
start method findById(1)
this script -> @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/addressInsert.sql") inserted data into tables country and address
table address will have next data:

id=5; apartment_number=1; city=Kiev e.t.c. 
id=6; apartment_number=4; city=Kiev e.t.c.
id=7; apartment_number=3; city=Bermuda e.t.c.
id=8; apartment_number=5; city=Azerbaijan e.t.c.

will error because Id with 1 removed before
after that this script -> @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDrop.sql") removed data from tables country and address

last test for method createTest()
this script -> @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/addressInsert.sql") inserted data into tables country and address
table address will have next data:

id=9; apartment_number=1; city=Kiev e.t.c. 
id=10; apartment_number=4; city=Kiev e.t.c.
id=11; apartment_number=3; city=Bermuda e.t.c.
id=12; apartment_number=5; city=Azerbaijan e.t.c.
id=13; apartment_number=89; city=Rio de Janeiro e.t.c.

will error because I need Id with 5 removed before
after that this script -> @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDrop.sql") removed data from tables country and address

Conclusion:
method findAllTest() = OK passed, will green
method findById(1) = error, will red
method createTest() = error, will red